README.md

```diff
@@ -1 +1,24 @@
-# actions
\ No newline at end of file
+## GitHub Actions Maintainer 
+
+The GitHub Actions Maintainer provides a set of GitHub Actions to make maitaining project easier.
```

Comment:
```suggestion
The GitHub Actions Maintainer provides a set of GitHub Actions to make maintaining projects easier.
```
What projects?

Reply:
The projects that will use the set of actions from this repository.